### ISSUE TITLE
hello!I would like to ask you for the accuracy of your code on GVCNN

### ISSUE THREAD

**Issue by GuoZLH** [waxnkw/gvcnn-pytorch#3]
I'm very interested in your work,a perfect code,and then I use it in modelnet40 ,but it often breaks,because the scores from fc will be very low  approaching 0.And this results in grouping only one group.
   can you tell me your accuracy on modelnet40.

          ----------------------good luck with you

**Comment by waxnkw** [comment]
Thanks for your appreciation.  I did not test it on modelnet40. I actually used GVCNN as a tool to take part in SHREC2019, a contest about 3D retrieval and get about 97% on that dataset. So, I did not get the acc on modelnet40.

 However, when I use the model on the contest, I have also met this problem.  So, I add a softmax layer after fc which can normalize the scores to 0~1 and ensure the summation of them is 1. So when some scores are approaching 0, it can avoid the bad BP which may cause the params are set to NaN. 

So what is the kind of your break, does it cause a run time exception or the params of the model are updated abnormally?
